### PR TITLE
tweak(TransferAmountWindow): QoL stuff

### PR DESCRIPTION
**Pull request by Brandon-Huu**
## About the PR

Disabled the apply button when the custom transfer amount is below or above the allowed bounds.
Show the allowed bounds on the UI.

## Why / Balance

QoL

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

Before:
image

After: 

**Disabled**
image
image
image

**Enabled**
image
image
image

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Brandon-Huu** [comment]
Wrong repository... 
I'll just make draft for now

**Comment by Brandon-Huu** [comment]
For some reason the min and max transfer amounts are returning values 100x larger than they should be

**Comment by Brandon-Huu** [comment]
Done

File: Content.Client/Chemistry/UI/TransferAmountWindow.xaml.cs [hunk 1]
```diff
@@ -21,10 +20,10 @@ public TransferAmountWindow()
 
         public void SetBounds(int min, int max)
```
Comment (deltanedas) [inline Content.Client/Chemistry/UI/TransferAmountWindow.xaml.cs:20]: im 99% sure its because casting fixedpoint2 to int isnt rounding down

Comment (Brandon-Huu) [reply Content.Client/Chemistry/UI/TransferAmountWindow.xaml.cs:20]: https://github.com/space-wizards/space-station-14/blob/cad1204a6da32fc5b75c1d7478fd8ab13f5f314a/Content.Client/Chemistry/UI/TransferAmountBoundUserInterface.cs#L29-L30

I solved the 100x value problem by just dividing the fixed point by 100.

image

Comment (Brandon-Huu) [reply Content.Client/Chemistry/UI/TransferAmountWindow.xaml.cs:20]: https://github.com/space-wizards/space-station-14/blob/master/Content.Shared/FixedPoint/FixedPoint2.cs#L196-L199

Oh I see, I shouldn't be using .Value

I should be using .Int()

**Pull request closed without merging**